Makefile

```diff
@@ -69,8 +69,9 @@ CONVERSION_GEN_VER := v0.22.2
 CONVERSION_GEN_BIN := conversion-gen
 CONVERSION_GEN := $(TOOLS_BIN_DIR)/$(CONVERSION_GEN_BIN)-$(CONVERSION_GEN_VER)
 
+ENVSUBST_VER := latest
```

Comment:
there isn't a release but we may want to pin to the same version that capi uses to avoid any differences that might occur between ci and tilt setups: https://github.com/kubernetes-sigs/cluster-api/blob/6eb2002ed15f73570ed14c83daceb55d18b2f939/hack/tools/go.mod#L7